src/kernel/include/inv.h

```diff
@@ -309,8 +309,6 @@ sret_ret(struct thread *thd, struct pt_regs *regs, struct cos_cpu_local_info *co
 	}
 
 	pgtbl_update(ci->pgtbl);
-	/* Set 2/3 return values into esi and edi */
-	__userregs_setretvals(regs, 0, thd->tid, 0);
```

Comment:
@gparmer I wasn't sure why this exists in `SRET` path, so I took it out when I was extending `__userregs_setretvals` to return 3 values. I may be missing something, even `cos_sinv` API doesn't have additional output parameters. 
Is there some gap between the intention behind the `cos_sinv` API and actual implementation?